Resources/Prototypes/_CM14/Entities/Structures/Machines/Lathe/lathe.yml

```diff
@@ -0,0 +1,114 @@
+﻿- type: entity
+  parent: [ BaseMachinePowered, ConstructibleMachine ]
+  id: CMAutoLathe
+  name: autolathe
+  description: It produces items using metal and glass.
+  components:
+  - type: Sprite
+    sprite: _CM14/Structures/Machines/autolathe.rsi
+    layers:
+    - state: autolathe_n
+      map: ["enum.LatheVisualLayers.IsRunning"]
+    - state: autolathe
+      shader: unshaded
+      map: ["enum.PowerDeviceVisualLayers.Powered"]
+    - state: autolathe_t
+      map: ["enum.WiresVisualLayers.MaintenancePanel"]
+  - type: Appearance
+  - type: WiresVisuals
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeAabb
+          bounds: "-0.5,-0.5,0.5,0.5"
+        density: 190
+        mask:
+        - MachineMask
+        layer:
+        - MachineLayer
+  - type: Lathe
+    idleState: autolathe
+    runningState: autolathe_n
+    staticRecipes:
+    - CMBucket
+#    - Janitorial bucket (10000 metal)
+#    - Flashlight (62 metal, 25 glass)
+#    - Extinguisher (112 metal)
+    - CMCrowbar
+    - CMMultitool
+    - CMTRayScanner
+    - CMWelderSmall
+    - CMScrewdriver
+    - CMWirecutter
+    - CMWrench
+#    - Mop (137 metal)
+#    - Radio headset (93 metal)
+#    - Shortwave radio (93 metal, 31 glass)
+#    - Welding mask (3750 metal, 1250 glass)
+#    - Steel sheets (3750 metal)
```

Comment:
remove this

Reply:
Done in [`6c04fa0` (#2085)](https://github.com/CM-14/CM-14/pull/2085/commits/6c04fa01a492f738e781657d72b8696c7c3582e4)